build/cqs.tsv

```diff
@@ -170,6 +181,7 @@ org.scalatra:scalatra-swagger_2.11	2.3.0	compile
 org.slf4j:slf4j-api	1.7.5	compile
 org.slf4j:slf4j-log4j12	1.7.5	compile
 org.tukaani:xz	1.0	compile
+org.xerial:sqlite-jdbc	3.8.11.2	compile
```

Comment:
this might be something we could exclude - it's used for persistence to disk (but might be used for other things too).